rfc#00002.md

```diff
@@ -0,0 +1,127 @@
+# RFC00002: Trust domain introduction
+- Authors: [Mike Bursell](mike@p2ptrust.org)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-13
+- Status Note: under discussion
+- Start Date: 2020-02-11
+- Tags: trust
+
+## Summary
+
+Trust domains are a key concept within Enarx.  This RFC provides an
+introduction to trust domains.
+
+## Motivation
+
+In order to explain the motivations behind Enarx, our design principles
+and aims, we need to be able to explain what trust domains are.  This RFC
+attempts to provide an introduction.
+
+## Tutorial
+
+Before considering trust, we need to consider why it is relevant
+to the issue of TEEs and Enarx.  The fundamental concept behind
+Enarx is that it allows applications - workloads - to be run on
+machines - hosts - which are not under the control of the owner
+of the workload.  The workload owner does not trust the host or
+the host owner.  But does this mean?
+
+Trust, in the human sphere, can be thought of the expectation one
+person - the trustor - has that another person - the trustee - will
```

Comment:
~~person - the trustor - has that another person - the trustee - will~~ -> person, the trustor, has that another person, the trustee, will

Reply:
Nope.

Reply:
No.